app/experimenter/experiments/admin/nimbus.py

```diff
@@ -36,6 +37,16 @@ class NimbusExperimentChangeLogInlineAdmin(admin.TabularInline):
 
 class NimbusExperimentAdminForm(forms.ModelForm):
     channel = forms.ChoiceField(choices=NimbusExperiment.Channel.choices)
+    public_description = forms.CharField(required=False, widget=forms.Textarea())
+    firefox_min_version = forms.ChoiceField(
+        choices=NimbusExperiment.Version.choices, required=False
+    )
+    channel = forms.ChoiceField(choices=NimbusExperiment.Channel.choices, required=False)
+    primary_outcomes = SimpleArrayField(forms.CharField(), required=False)
+    secondary_outcomes = SimpleArrayField(forms.CharField(), required=False)
```

Comment:
Can outcomes be an HTML multiselect that populates with slugs?

Reply:
do we need any validation on this? I can enter any value and save

Reply:
No unfortunately we can't validate it because the outcome list and the outcomes stored aren't required to be consistent, see: https://github.com/mozilla/experimenter/tree/main/app/experimenter/outcomes#consistency